argus/backend/service/build_system_monitor.py

```diff
@@ -68,17 +73,42 @@ def check_filter(self, group_name: str) -> bool:
 
 
 class JenkinsMonitor(ArgusTestsMonitor):
+
+    BUILD_SYSTEM_FILTERED_PREFIXES = [
+        "byo",
```

Comment:
why filtering byo jobs ? it's can be quite useful to look at them with argus... (i.e. the SCT ones)

Reply:
Ditto